packages/react-data-grid-addons/src/__tests__/Grid.spec.js

```diff
@@ -104,6 +104,76 @@ describe('Grid', function() {
     }, events));
   });
 
+  // Set of tests for the props that defined the height of our rows
+  describe('when defininig heigths on props', function() {    
+    describe('for defaults props', function() {
+      beforeEach(function() {
+        const ToolBarStub = new StubComponent('Toolbar');
+        this.component = this.createComponent({ toolbar: <ToolBarStub /> }).node;
+        this.toolbarInstance = TestUtils.findRenderedComponentWithType(this.component, ToolBarStub);
+        this.toolbarInstance.props.onToggleFilter();
```

Comment:
This is fine. In future though, take a look at http://airbnb.io/enzyme/
You'll find the tests much easier to write, and its what we should be using for React component tests going forward